values.yaml

```diff
@@ -241,3 +241,21 @@ initContainers: []
 ## @param sidecars extra containers running alongside the vaultwarden instance
 ##
 sidecars: []
+
+## @section Extra Configuration
+##
+
+## @param nodeSelector Node labels for pod assignment
+## Ref: https://kubernetes.io/docs/user-guide/node-selection/
```

Comment:
Please review the URL, it appears to be broken at the moment. Thanks.

Reply:
@guerzon thank you for your attention to details. I've fix it